src/logic/resources-status.ts

```diff
@@ -0,0 +1,39 @@
+import { AppComponents } from '../types'
+import os from 'node-os-utils'
```

Comment:
How is this different to import "os"? The load and memory usage are natively exposed by node

Reply:
Hello @menduz, using built-in os library was my first try to avoid installing a new package just for this. Also, I've tried to pull these metrics from prometheus but discarded the approach for the back-and-forth it produced.

I ended up using 'node-os-utils' because 'os' gives me different values than the ones displayed in Activity Monitor of my computer. Maybe I did the calculations wrongly. You may find more details about the differences I found trying both libraries below:

```json
{
  os: {
    memory: {
      memLoad: 99.63665008544922,
      calculation: '([os.totalmem() - os.freemem()] as usedMem * 100) / os.totalmem()'
    },
    cpu: { cpuLoad: 6.69580078125, calculation: 'os.loadavg()[0]' }
  },
  'node-os-utils': {
    memory: {
      memLoad: 83.506103515625,
      calculation: '(os.mem.used().usedMemMb * 100) / os.mem.used().totalMemMb'
    },
    cpu: { cpuLoad: 19.18, calculation: 'os.cpu.usage()' }
  }
}
```

The values exposed on `node-os-utils` correspond with the values being shown in the Activity Monitor. I added the calculations because maybe this is because I miss-calculating the values from `os` built-in library.

Reply:
The thing is that I wanted to avoid installing a new package, so new approaches that avoid this are more than welcome 🙏 .

Reply:
The os measurements are different because it gives you three numbers. Which are time windows of 1, 5 and 15 minutes (moving averages).

This is the most significant measurement you may use to flatten spikes in cpu load. I think the 1 or 5 minutes measurements will be more reliable than the immediate measurement of the activity monitor, that one is hardly measurable and usable

https://nodejs.org/api/os.html#osloadavg

Reply:
Oh okay, I'll use and trust the 1m CPU measurement, but still I need to figure out what's going on with the memory measurement since it always returns 99%. I'll double-check those numbers. Maybe it is expected but I'm not seeing any spike in my Activity Monitor related to memory.

Reply:
Never mind, I've tried this snipper on peer-testing-2 by running:
```bash
docker exec -it <CONTAINER> /bin/bash -c "node - <<EOF
const os = require('os')
const totalMem = os.totalmem()
const freeMem = os.freemem()
const usedMem = totalMem - freeMem
const memLoad = (usedMem * 100) / totalMem
console.log({ totalMem, freeMem, usedMem, memLoad})
EOF"
```

And printed:
```js
{
  totalMem: 16592396288,
  freeMem: 10954158080,
  usedMem: 5638238208,
  memLoad: 33.980855508361394
}
```